apps/front-client/src/features/profile/components/Atoms/AlertMessage.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+import { palette, typography } from '@hetic-saline-royale-academy/kit-ui';
+import { Typography } from '@mui/material';
+import { PropsWithChildren } from 'react';
+
+const Alert = styled(Typography)`
+  color: ${palette.error[500]};
+  ${typography.xs.medium};
+`;
+
+const AlertMessage = ({ children }: PropsWithChildren) => {
+  return <Alert>{children}</Alert>;
+};
+
```

Comment:
Alors c'est pas faux mais pro tips tu peux faire juste ça :)

```suggestion
import styled from '@emotion/styled';
import { palette, typography } from '@hetic-saline-royale-academy/kit-ui';
import { Typography } from '@mui/material';

const Alert = styled(Typography)`
  color: ${palette.error[500]};
  ${typography.xs.medium};
`;

export default Alert;
```